projects/browser-extension/vue-app/src/views/facebook.view.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <div class="facebook">
-    <switch-buttons
+    <!-- <switch-buttons
```

Comment:
Is this documented somewhere? Maybe put it in Trello.

Reply:
It's in Trello ;)